Gemfile.jruby-1.9.lock.release

```diff
@@ -125,7 +125,7 @@ GEM
     jls-lumberjack (0.0.26)
       concurrent-ruby
     jmespath (1.3.1)
-    jrjackson (0.4.4-java)
```

Comment:
This should be held back at 0.4.4 until we get the version of Jackson updated , else we get an annoying (but harmless) warning about version mismatch.